metrics/mixin/config.libsonnet

```diff
@@ -5,9 +5,12 @@
 
     // Duration to raise various Alerts
     clusterObjectStoreStateAlertTime: '15s',
+    ocsMirrorDaemonStatusAlertTime: '1m',
+    ocsPoolMirroringImageHealthAlertTime: '1m',
```

Comment:
Very less time alert time may raise a false alert when image state and daemon is in some temporary health. better lets wait for 1m